libz-rs-sys/src/gz.rs

```diff
@@ -866,13 +873,9 @@ pub unsafe extern "C-unwind" fn gzread(file: gzFile, buf: *mut c_void, len: c_ui
 
     // Check that the requested number of bytes can be represented by the return type.
     if c_int::try_from(len).is_err() {
+        const MSG: &str = "request does not fit in an int";
```

Comment:
just as a note, `const MSG: &str = "some literal"` is the same as `let msg = "some literal"`. String literals always get a static lifetime and are stored in the directly in the binary, just like constants.